client/main.lua

```diff
@@ -35,19 +28,19 @@ end)
 
 AddEventHandler('onResourceStart', function(resourceName)
     if GetCurrentResourceName() ~= resourceName or not isLoggedIn then return end
-    hasFitbit = fitbitCheck(QBX.PlayerData.items)
+    hasFitbit = exports.ox_inventory:Search('count', 'fitbit') >= 1
 end)
 
 RegisterNetEvent('QBCore:Client:OnPlayerLoaded', function()
-    hasFitbit = fitbitCheck(QBX.PlayerData.items)
+    hasFitbit = exports.ox_inventory:Search('count', 'fitbit') >= 1
 end)
 
 RegisterNetEvent('QBCore:Client:OnPlayerUnload', function()
     hasFitbit = false
 end)
 
 RegisterNetEvent('QBCore:Player:SetPlayerData', function()
```

Comment:
The event itself should be an ox_inventory change inventory invent

Reply:
what event is this?

Reply:
https://overextended.dev/ox_inventory/Events/Client#ox_inventoryitemcount

Reply:
And how would this properly be incorporated across these three handlers with the item check

Reply:
If you want to know whether the player has a Fitbit or not, listen to inventory changes. The event tells you every time an item changes its count. So write an if condition to check if the item that changes count is 'Fitbit' and the count > 0